pmm-app/tests/ia/pages/api/channelsAPI.js

```diff
@@ -60,6 +64,7 @@ module.exports = {
     const body = { channel_id: channelId };
     const resp = await I.sendPostRequest('v1/management/ia/Channels/Remove', body, headers);
 
-    assert.ok(resp.status === 200, `Failed to remove channel with channel_id=${channelId}`);
+    assert.ok(resp.status === 200, `Failed to remove channel with channel_id "${channelId}".
+     Response message is "${resp.data.message}"`);
```

Comment:
```suggestion
    assert.ok(
      resp.status === 200, 
      `Failed to remove channel with channel_id "${channelId}". Response message is "${resp.data.message}"`
    );
```
This is how to approach similar long lines :)